### YamlMime:ManagedReference
items:
- uid: System.Security.XmlSyntaxException
  id: XmlSyntaxException
  children:
  - System.Security.XmlSyntaxException.#ctor
  - System.Security.XmlSyntaxException.#ctor(System.Int32)
  - System.Security.XmlSyntaxException.#ctor(System.String)
  - System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  - System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException
  fullName: System.Security.XmlSyntaxException
  type: Class
  summary: "XML 구문 분석에 구문 오류가 있을 때 throw 되는 예외입니다. 이 클래스는 상속 될 수 없습니다."
  remarks: "기본값을 사용 하 여 XmlSyntaxException <xref:System.Object.Equals%2A>구현을 참조 일치를 지원 합니다.</xref:System.Object.Equals%2A>       목록이 XmlSyntaxException의 인스턴스에 대 한 초기 속성 값에 대 한 생성자를 참조 하십시오."
  syntax:
    content: >-
      [System.Runtime.InteropServices.ComVisible(true)]

      public sealed class XmlSyntaxException : SystemException
  inheritance:
  - System.Exception
  - System.SystemException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor
  id: '#ctor'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> 클래스 기본 속성을 사용 합니다."
  remarks: "다음 표에서 <xref:System.Security.XmlSyntaxException>안내 하십시오.</xref:System.Security.XmlSyntaxException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; 구문이 잘못 되었습니다. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException ();
    parameters: []
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  id: '#ctor(System.Int32)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> 예외가 발견 된 줄 번호를 사용 하 여 클래스입니다."
  remarks: "XML 스트림의 줄 번호는 줄 바꿈 문자가 발견 될 때마다 업데이트 됩니다. 번호는 첫 번째 줄에 대해 1로 시작합니다.       다음 표에서 <xref:System.Security.XmlSyntaxException>안내 하십시오.</xref:System.Security.XmlSyntaxException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; 줄에 잘못 된 구문이 `lineNumber`. &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "XML 구문 오류가 검색 되었습니다 XML 스트림의 줄 번호입니다."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> 지정된 된 오류 메시지를 사용 하 여 클래스입니다."
  remarks: "다음 표에서 <xref:System.Security.XmlSyntaxException>안내 하십시오.</xref:System.Security.XmlSyntaxException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>| 오류 메시지 문자열입니다. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message);
    parameters:
    - id: message
      type: System.String
      description: "예외에 대 한 이유를 설명 하는 오류 메시지입니다."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  id: '#ctor(System.Int32,System.String)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> 된 지정된 된 오류 메시지 및 예외가 발견 된 줄 번호입니다."
  remarks: "XML 스트림의 줄 번호는 줄 바꿈 문자가 발견 될 때마다 업데이트 됩니다. 번호는 첫 번째 줄에 대해 1로 시작합니다.       다음 표에서 <xref:System.Security.XmlSyntaxException>안내 하십시오.</xref:System.Security.XmlSyntaxException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| `null`.|   | <xref:System.Exception.Message%2A>|&quot; 줄에 잘못 된 구문이 `lineNumber`  -  `message` &quot;|</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (int lineNumber, string message);
    parameters:
    - id: lineNumber
      type: System.Int32
      description: "XML 구문 오류가 검색 되었습니다 XML 스트림의 줄 번호입니다."
    - id: message
      type: System.String
      description: "예외에 대 한 이유를 설명 하는 오류 메시지입니다."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Security.XmlSyntaxException
  langs:
  - csharp
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
  type: Constructor
  assemblies:
  - mscorlib
  namespace: System.Security
  summary: "새 인스턴스를 초기화는 <xref href=&quot;System.Security.XmlSyntaxException&quot;> </xref> 지정된 된 오류 메시지와 해당 예외의 원인인 내부 예외에 대 한 참조를 사용 하 여 클래스입니다."
  remarks: "이전 예외의 직접적인 결과에 이전 예외에 대 한 참조가 들어 있어야 throw 되는 예외는 <xref:System.Exception.InnerException%2A>속성.</xref:System.Exception.InnerException%2A> <xref:System.Exception.InnerException%2A>속성은 생성자에 전달 되는 동일한 값을 반환 하거나 `null` 경우는 <xref:System.Exception.InnerException%2A>생성자에 내부 예외 값을 제공 하지 않는.</xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       다음 표에서 <xref:System.Security.XmlSyntaxException>안내 하십시오.</xref:System.Security.XmlSyntaxException> 인스턴스에 대 한 초기 속성 값을 보여 줍니다.      | 속성 | 값 |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| 내부 예외를 참조 합니다. |   | <xref:System.Exception.Message%2A>| 오류 메시지 문자열입니다. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public XmlSyntaxException (string message, Exception inner);
    parameters:
    - id: message
      type: System.String
      description: "예외에 대 한 이유를 설명 하는 오류 메시지입니다."
    - id: inner
      type: System.Exception
      description: "현재 예외의 원인인 예외입니다. 경우는 `inner` 매개 변수가 않습니다 <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;> </xref>에서 현재 예외가 발생 한 <xref uid=&quot;langword_csharp_catch&quot; name=&quot;catch&quot; href=&quot;&quot;> </xref> 내부 예외를 처리 하는 블록입니다."
  overload: System.Security.XmlSyntaxException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.SystemException
  isExternal: false
  name: System.SystemException
- uid: System.Security.XmlSyntaxException.#ctor
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException()
  nameWithType: XmlSyntaxException.XmlSyntaxException()
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException()
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32)
- uid: System.Int32
  parent: System
  isExternal: true
  name: Int32
  nameWithType: Int32
  fullName: System.Int32
- uid: System.Security.XmlSyntaxException.#ctor(System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Security.XmlSyntaxException.#ctor(System.Int32,System.String)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(Int32,String)
  nameWithType: XmlSyntaxException.XmlSyntaxException(Int32,String)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(Int32,String)
- uid: System.Security.XmlSyntaxException.#ctor(System.String,System.Exception)
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException(String,Exception)
  nameWithType: XmlSyntaxException.XmlSyntaxException(String,Exception)
  fullName: System.Security.XmlSyntaxException.XmlSyntaxException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Security.XmlSyntaxException.#ctor*
  parent: System.Security.XmlSyntaxException
  isExternal: false
  name: XmlSyntaxException
  nameWithType: XmlSyntaxException.XmlSyntaxException
